### ISSUE TITLE
Text cursor not displaying correctly on iOS browser

### ISSUE THREAD

**Issue by kro4ever** [mattermost/mattermost-server#8187]
**Summary**
Text cursor not showing on the right position of text box or not showing at all
**Steps to reproduce**
1. Go to mattermost site on iPhone 7
2. Access your account
3. Open on browser, not app
4. Go to a channel and reply to a message
5. Verify that the text box doesn't include the text cursor or the text cursor is misplaced. 
**Expected behavior**
Once user clicks on reply button, text cursor should display at the beginning of the text box field. 
**Observed behavior**
First behavior, text cursor was misplaced. 
Second behavior, text cursor dissappeared. 
![80224](https://user-images.githubusercontent.com/36074255/35733034-c1691116-07e9-11e8-8b81-f5fb7f19e5d2.jpg)
![67254](https://user-images.githubusercontent.com/36074255/35733068-e2e303ba-07e9-11e8-8a11-11627cd097ce.jpg)

**Comment by lindy65** [comment]
Hi @kro4ever,

Thank you for your feedback,

On https://oss.mattermost.com I could not reproduce the issue of the cursor being misplaced but confirm that there is no cursor in the text input box after clicking on "reply" to a message.

oss.mattermost.com is running v4.6.1 (rc1) so I also tested on our [nightly build server](https://pre-release.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676) running off the latest master and got the same results - no misplaced cursor but also no cursor in the text input box after selecting "reply" to a message.

I've created [this improvement ticket](https://mattermost.atlassian.net/browse/PLT-8689) to fix the issue.

To be noted is that I tested on Android 7 as I do not have an iPhone.
